wdl/VisualizeCnvs.wdl

```diff
@@ -0,0 +1,139 @@
+version 1.0
+
+import "Structs.wdl"
+
+# Plots CNV depth profiles across batches
+
+workflow VisualizeCnvs {
+  input{
+    File? vcf  # Option 1
+    File? bed  # Option 2, columns: chrom,start,end,name,svtype,samples
```

Comment:
Can both be given? if not, would not using one variable/arg and deciding based on the extension of the provided value whether is bed (compressed or decompressed) or VCF be simpler interface? 

On a similar note, since you are supporting zipped and unzipped `bed` would it be useful to support compressed and raw `vcf` as well?

Reply:
That's a good idea - I've implemented it as a single variable. Decompressed vcfs are very uncommon so I won't include support for those though.